test/unit/org/apache/cassandra/index/sai/utils/RangeConcatIteratorTest.java

```diff
@@ -335,37 +335,6 @@ public void testIntersectionOfConcat()
         assertEquals(convert(1L, 3L, 5L, 7L, 9L), convert(buildIntersection(concatA, concatB)));
     }
 
-    @Test
-    public void testConcatOnError()
-    {
-        assertOnError(buildOnErrorA(CONCAT, arr(1L, 2L, 3L), arr(4L, 5L, 6L)));
-        assertOnError(buildOnErrorB(CONCAT, arr( 1L, 2L, 3L), arr(4L)));
-    }
-
-    @Test
-    public void testConcatOfUnionsOnError()
-    {
-        RangeIterator unionA = buildUnion(arr( 1L, 2L, 3L), arr( 4L));
-        RangeIterator unionB = buildOnErrorB(UNION, arr( 6L), arr( 8L, 9L));
-        assertOnError(buildConcat(unionA, unionB));
-
-        unionA = buildOnErrorA(UNION, arr( 1L, 2L, 3L), arr( 4L));
-        unionB = buildUnion(arr( 5L), arr( 5L, 6L));
-        assertOnError(buildConcat(unionA, unionB));
-    }
-
-    @Test
-    public void testConcatOfIntersectionsOnError()
-    {
-        RangeIterator intersectionA = buildOnErrorA(INTERSECTION, arr( 1L, 2L, 3L), arr( 2L, 3L, 4L));
-        RangeIterator intersectionB = buildIntersection(arr( 6L, 7L, 8L), arr( 7L, 8L, 9L));
-        assertOnError(buildConcat(intersectionA, intersectionB));
-
-        intersectionA = buildIntersection(arr( 1L, 2L, 3L), arr( 2L, 3L, 4L));
-        intersectionB = buildOnErrorB(INTERSECTION, arr( 6L, 7L, 8L, 9L, 10L), arr(  7L, 8L, 9L));
-        assertOnError(buildConcat(intersectionA, intersectionB));
-    }
-
```

Comment:
Why remove those tests?

Reply:
this is testing the "injected" `shouldThrow` in LongArray, which exists to test async TPC paths.  TPC doesn't exist in CC so it doesn't add value.